### ISSUE TITLE
Cannot run program "ssh://kogli2000@35.198.136.114:22/home/kogli2000/miniconda3/envs/ai/bin/python" (in directory "/home/jamshaid/DeepTraffic"): error=2, No such file or directory

### ISSUE THREAD

**Issue by jamshaid120** [AlexanderKoch-Koch/DeepTraffic#1]
I am having this issue while running the project in Linux
Cannot run program "ssh://kogli2000@35.198.136.114:22/home/kogli2000/miniconda3/envs/ai/bin/python" (in directory "/home/jamshaid/DeepTraffic"): error=2, No such file or directory

**Comment by AlexanderKoch-Koch** [comment]
I think that\s because I forgot to add my .idea folder to the gitignore. And you used the run config. You can just run "python3  DeepTraffic.py" in the console. But I think it still doesn't run. It's an old project and I don't work on it anymore.

**Comment by jamshaid120** [comment]
right. It also contains a lot of errors. It tries to convert **b ' '** into float in a class which is a bug at all.
